src/server/service/diagnostics.rs

```diff
@@ -666,14 +690,14 @@ mod log {
             let mut req = SearchLogRequest::default();
             req.set_start_time(timestamp("2019/08/23 18:09:54.387 +08:00"));
             req.set_end_time(std::i64::MAX);
-            req.set_level(LogLevel::Warn);
-            req.set_pattern("test-filter".to_string());
+            req.set_levels(vec![LogLevel::Warn.into()].into());
```

Comment:
@nrc Please take a look about repeated enum. Looks like repeated enum becomes `Vec<i64>` instead of `Vec<SomeEnum>`, which should be a mistake.